### PR TITLE
Change comment assignee to multiselect

### PR DESCRIPTION
**Pull request by AdityaKhatri**
Depends on: https://github.com/the-deep/server/pull/476

### CODE REVIEW

File: src/components/general/Navbar/Notifica/Notifications/items/EntryCommentItem/index.js [hunk 1]
```diff
@@ -102,6 +102,12 @@ export default class EntryCommentItem extends React.PureComponent {
                 'entryCommentReplyAdd',
                 stringParams,
             );
+        } else if (notificationType === 'entry_comment_assignee_change') {
+            notificationText = _ts(
+                'entryComments.notifications',
+                'entryCommentAdd',
```
Comment (tnagorra) [inline src/components/general/Navbar/Notifica/Notifications/items/EntryCommentItem/index.js:108]: Should this not have a different messag than entryCommentAdd?

File: src/views/Entries/LeadGroupedEntries/Entry/index.js [hunk 2]
```diff
@@ -222,7 +222,7 @@ export default class Entry extends React.PureComponent {
                                 <EntryCommentModal
                                     entryServerId={entryId}
                                     onCommentsCountChange={this.handleCommentsCountChange}
-                                    defaultAssignee={createdBy}
+                                    defaultAssignees={[createdBy]}
```
Comment (tnagorra) [inline src/views/Entries/LeadGroupedEntries/Entry/index.js]: We shouldn't create new element on render.

**Review by tnagorra — COMMENTED** [review]
Some minor problems.

**Review by tnagorra — CHANGES_REQUESTED** [review]
Some minor problems!

File: src/components/general/Navbar/Notifica/Notifications/items/EntryCommentItem/index.js (hunk 1 above)
Comment (AdityaKhatri) [reply src/components/general/Navbar/Notifica/Notifications/items/EntryCommentItem/index.js:108]: No it's fine, it will follow the same syntax.

**Review by tnagorra — APPROVED** [review]

**Pull request merged**